src/main/Config.cpp

```diff
@@ -1226,6 +1241,14 @@ Config::validateConfig(ValidationThresholdLevels thresholdLevel)
                                   UNSAFE_QUORUM ? 1 : 51);
         throw std::invalid_argument("Invalid QUORUM_SET");
     }
+
+    if (METADATA_OUTPUT_FILE_DESCRIPTOR != -1 &&
+        METADATA_OUTPUT_NAMED_PIPE != "")
+    {
+        LOG(FATAL) << "Can only set one of METADATA_OUTPUT_NAMED_PIPE and"
```

Comment:
we could also only have one setting that can be either a path or an integer?

Reply:
I think this is not ideal: it's weird to the user (precludes integer-valued filenames, granted, unlikely) and requires weird implementation (a config-value visitor I guess? something that precludes accessing the string as a string when it's "really" an integer..)

Reply:
Fixed: went with "pathname which can be a pipe or a file, or 'fd:N' which opens the N file descriptor". User can deal with living without files named "fd:N".